templates/sfn/ingest_run_workflow_sfn_definition.json.tpl

```diff
@@ -122,10 +122,14 @@
               "StringEquals": "Succeeded"
             }
           ],
-          "Next": "Map over each assetId and reconcile"
+          "Next": "End Workflow and return WorkflowResult"
         }
       ],
       "Default": "Wait 5 minutes before getting status"
+    },
+    "End Workflow and return WorkflowResult": {
+      "Type": "Pass",
+      "End": true
```

Comment:
I think we can save a state transition by making this a `Succeed` state instead of a `Pass`.

https://docs.aws.amazon.com/step-functions/latest/dg/state-succeed.html

Reply:
But can a `Succeed` state pass on its Output?

Reply:
Good question! It ends the execution so I'd assume it does, in the same way that `"End": true` does in the background.